frontend/src/shared/components/tasque-menu-dropdown/menu-dropdown.component.ts

```diff
@@ -44,6 +44,11 @@ export class MenuDropdownComponent implements OnInit, ControlValueAccessor {
   @Input() public hasAvatar = false;
   @Output() avatarClicked = new EventEmitter();
 
+  @Input() headerLabel: string;
```

Comment:
please add comments to inputs and outputs about their purpose and expected data

Reply:
Done